compose.yaml

```diff
@@ -1,138 +0,0 @@
-services:
```

Comment:
Nothing in this file should be changed. This is the kind of thing you should be looking out for in your self-review.